### ISSUE TITLE
Enhancement request: Multiple locations for each session

### ISSUE THREAD

**Issue by heldertb** [AHAAAAAAA/PokemonGo-Map#2468]
I installed the software on a server and I'm using it along with some friends of mine. But you can only add 1 location, so if a few other people use it, the marker starts jumping around every time you refresh, is there a possibility to change this and make multiple locations possible?
I'm not very familiar with python, otherwise I'd make a pull request...

**Comment by ecrouch** [comment]
If you want to search multiple locations I think you'll need to run several accounts simultaneously and concatenate the data into one python server that has 'seats' that can be taken up to a maximum equal to the number of accounts you have running. I have virtually no python experience but a workaround that's annoying yet effective is to run several servers individually, on different ports, and use something like ngrok or heroku to supply unique urls that can be sent to individuals. That last solution is currently working for me on my desktop at home. I only run 2 accounts though.

**Comment by heldertb** [comment]
Good idea, but I'd like to see this native too. I know this is possible and not that hard to program. Just parse locations through a session in your browser and do not put 1 location in the backend, scan multiple locations with multiple threads...
